releasenotes/notes/fix-numpy-indices-marginal-dist-45889e49ba337d84.yaml

```
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Fixed an issue with the :func:`~.marginal_distribution` function when
```

I'm trying to make this sentence not be a run-on.  There are a few acceptable ways one could do this.  Here's one.

```suggestion
    Fixed an issue with the :func:`~.marginal_distribution` function: when
```